### PR TITLE
DOCSUP-7502: Document the database engine Replicated

### PR DESCRIPTION
**Pull request by adevyatova**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation for #10485

### CODE REVIEW

File: docs/en/engines/database-engines/replicated.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
+# Replicated {#replicated}
```
Comment (tavplubix) [inline docs/en/engines/database-engines/replicated.md:2]: Maybe we should add a note that Replicated engine is experimental

File: docs/en/engines/database-engines/replicated.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Supports replication of metadata via DDL log being written to ZooKeeper and executed on all of the replicas for a given database.
+One Clickhouse server can have multiple replicated databases running and updating at the same time.
```
Comment (tavplubix) [inline docs/en/engines/database-engines/replicated.md:5]: It's true, one server can have multiple different replicated databases, but it cannot have multiple replicas of one replicated database (we should notice it).

File: docs/en/engines/database-engines/replicated.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## Creating a Database {#creating-a-database}
+``` sql
+    CREATE DATABASE testdb ENGINE = Replicated('zoo_path', 'replica_name')
```
Comment (tavplubix) [inline docs/en/engines/database-engines/replicated.md:9]:
```suggestion
    CREATE DATABASE testdb ENGINE = Replicated('zoo_path', 'shard_name', 'replica_name') [SETTINGS ...]
```

File: docs/en/engines/database-engines/replicated.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+```
+
+**Engine Parameters**
+
+-   `zoo_path` — ZooKeeper path. The same ZooKeeper path corresponds to the same database.
+-   `replica_name` — Replica name. Replica names must be different for all replicas of the same database.
```
Comment (tavplubix) [inline docs/en/engines/database-engines/replicated.md:15]:
```suggestion
-   `shard_name` — Shard name. Database replicas are grouped into shards by `shard_name`.
-   `replica_name` — Replica name. Replica names must be different for all replicas of the same shard.
```

File: docs/en/engines/database-engines/replicated.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+Using this engine, creation of Replicated tables requires no ZooKeeper path and replica name parameters. Table's replica name is the same as database replica name. Table's ZooKeeper path is a concatenation of database's ZooKeeper path, `/tables/`, and `UUID` of the table.
```
Comment (tavplubix) [inline docs/en/engines/database-engines/replicated.md:17]: As for now it's incorrect, behavior is the same as with `Atomic` engine. If no arguments provided for `ReplicatedMergeTree` engine, then default arguments are used: `/clickhouse/tables/{uuid}/{shard}` and `{replica}` (can be changed by server settings `default_replica_path` and `default_replica_name`). Macro `{uuid}` is unfolded to table's uuid, `{shard}` and `{replica}` are unfolded to values from server config, not from database engine arguments.

But I should change it and make `ReplicatedMergeTree` use shard and replica name of `Replicated` database.

File: docs/en/operations/settings/settings.md [hunk 6]
```diff
@@ -3065,4 +3065,48 @@ SELECT
...
+Sets how long initial DDL query should wait for Replicated database to precess previous DDL queue entries in seconds.
+
+Possible values:
+
+-   Positive integer.
+-   0 — Unlimited.
+
+Default value: `300`.
+
+## database_replicated_ddl_output {#database_replicated_ddl_output}
```
Comment (tavplubix) [inline docs/en/operations/settings/settings.md:3090]: The setting is obsolete, it has been replaced with `distributed_ddl_output_mode` (#21535)

File: docs/en/operations/settings/settings.md [hunk 7]
```diff
@@ -3065,4 +3065,48 @@ SELECT
...
+Returns table with query execution status as a result of DDL query.
+
+Possible values:
+
+-   0 — Disabled.
+-   1 — Enabled.
+
+Default value: `1`.
+
+## background_replicated_schedule_pool_size {#background_replicated_schedule_pool_size}
```
Comment (tavplubix) [inline docs/en/operations/settings/settings.md:3101]: There is no such setting

**Comment by adevyatova** [comment]
@tavplubix Hi! Take a look, please!

**Review by tavplubix — APPROVED** [review]

**Pull request merged**